lua/dap.lua

```diff
@@ -678,6 +699,17 @@ function M.down()
   end
 end
 
+function M.goto_(line)
+  if session then
+    local source, col
+    if not line then
+      line, col = unpack(api.nvim_win_get_cursor(0))
```

Comment:
`col` here is 0-indexed. Is that also what the DAP expects for the gotoTargetsRequest?

Reply:
No. in the init request. You specified that column start at 1. I'llf fix that